doc/_static/mpl.css

```
@@ -185,3 +192,17 @@ div.wide-table table th.stub {
 .sdd.sd-dropdown .sd-card-header +.sd-card-body{
     --pst-sd-dropdown-color: none;
 }
+
+/* section-toc is a custom class that removes the page title from a toctree listing
+ * Example usage:
+ *
+ *     .. rst-class:: section-toc
+ *
+ *        .. toctree::
```

```suggestion
 *     .. rst-class:: section-toc
 *     .. toctree::
```
I would use the compact notation to make it more clear, these belong together.

Side-note: I've added a `:class:` option to `.. toctree` (https://github.com/sphinx-doc/sphinx/pull/12524), so that in future sphinx versions, we can simply use

```
.. toctree::
   :class: section-toc
```